doc/sphinx/how_to_parse.rst

```diff
@@ -218,3 +219,20 @@ Using ``xyz`` as the <OutputFileFormat> with Benzeneselenol.out, we obtain the f
     H     -2.7028960000   -2.1530750000    0.0036640000
     Se     1.8210800000   -0.0433780000   -0.0038760000
     H      2.0043580000    1.4100070000    0.1034490000
+
+ccframe
+-------
+
+This script creates complete tables of data from output files in some of the formats supported by pandas_.
+Since the pandas library is not a dependency of cclib, `it must be installed <https://pandas.pydata.org/pandas-docs/stable/install.html>`_ separately.
+
+.. _pandas: https://pandas.pydata.org/
+
+A complete data table can be parsed from many output files by following this format
+
+.. code-block:: bash
+
+    ccframe -O <OutputDest> <CompChemLogFile> [<CompChemLogFile>...]
+
+The argument for ``-O`` indicates the data file to be written and its extesion specifies the filetype (e.g. csv, hdf/hdf5, json/cjson, pickle/pkl, xlsx).
```

Comment:
I would remove CJSON from here. The JSON writer we have (via `ccwrite` outputs CJSON, but Pandas is going to generate regular JSON.

I will update the `ccwrite` docs to make it clear that it can't write regular JSON, only CJSON.

Reply:
This was changed in 8aa8fb2.